lib/vehicle.rb

```diff
@@ -5,14 +5,20 @@ class Vehicle
               :year,
               :make,
               :model,
-              :engine
+              :engine,
+              :registration_date
+
+  attr_accessor :plate_type
```

Comment:
The attr_accessor is technically a method, so you'll want to make sure you write tests for this method to make sure you can read and update the attribute.